SLO Primary Time Window (#16182)
DataDog/documentation: content/en/monitors/service_level_objectives/_index.md

### Configuration

1. On the [SLO status page][1], select **New SLO +**.
2. Define the source for your SLO. You can create an SLO from [metrics][3] or [monitors][4].

> 3. Set a target and a rolling time window (past 7, 30, or 90 days) for the SLO. It is recommended that you make the target stricter than your stipulated SLAs. If you configure more than one time window, select one to be the primary time window. This time window will be displayed on SLO lists. By default, the shortest time window will be selected. 

Reviewer: ```suggestion
3. Set a target and a rolling time window (past 7, 30, or 90 days) for the SLO. Datadog recommends you make the target stricter than your stipulated SLAs. If you configure more than one time window, select one to be the primary time window. This time window is displayed on SLO lists. By default, the shortest time window is selected. 
```
